### PR TITLE
Fix typo in syntax.md

### PR DESCRIPTION
**Pull request by magurotuna**
This might be a typo, so I fixed it.

### CODE REVIEW

File: docs/dev/syntax.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Points of note:
 * Trivia and non-trivia tokens are not distinguished on the type level.
 * Each token carries its full text.
 * The original text can be recovered by concatenating the texts of all tokens in order.
-* Accessing a child of particular type (for example, parameter list of a function) generally involves linerary traversing the children, looking for a specific `kind`.
+* Accessing a child of particular type (for example, parameter list of a function) generally involves linear traversing the children, looking for a specific `kind`.
```
Comment (bjorn3) [inline docs/dev/syntax.md:75]: I think the intended word is:

```suggestion
* Accessing a child of particular type (for example, parameter list of a function) generally involves linearly traversing the children, looking for a specific `kind`.
```

Comment (magurotuna) [reply docs/dev/syntax.md:75]: ahh, that's right, it'd be better to use adverb here. I'll fix it soon.

Comment (magurotuna) [reply docs/dev/syntax.md]: Done :)

**Pull request merged**